portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSettings.macro

```diff
@@ -10,6 +10,20 @@ definition {
 		Click(locator1 = "ACSettings#KEYWORD_SEND_BUTTON");
 	}
 
+	macro assertAddonsDetails {
+		for (var limits : list "${limitsIndividuals},${limitsPageViews}") {
+			var key_limits = StringUtil.replace("${limits}", ".", ",");
+
+			AssertElementPresent(locator1 = "ACUsage#USAGE_ADDONS_LIMITS");
+		}
```

Comment:
Hi @AfonsoHSantos , I would suggest you remove the String manipulation since once I suggested you pass the values using commas instead of periods.

Also, I would suggest you not use loops here to increase the code's legibility. So I would keep it with repeated asserts.

It is not so common or good practice to build lists with different parameters, when we are using lists, it is most common to use like this: 

```
assertAddonsDetails(
limitList = "5\,000,5\,000\,000");
```

Use `\,` to escape the commas that are considered the delimiter for lists in Poshi.